packages/Pagination/Pagination.jsx

```diff
@@ -171,9 +257,7 @@ Pagination.propTypes = {
   copy: PropTypes.oneOf(['en', 'fr']),
```

Comment:
I recommend marking this as `isRequired` so that copy is explicitly provided. This would be a breaking change.

This is out of scope, but if you wish to support multilingual, I recommend following the example in [Footnote](https://github.com/telus/tds-core/blob/878cccdb8d44ebea00a887b2efe62ac0217f4c74/packages/TermsAndConditions/Footnote/Footnote.jsx#L327-L338).